client/lib/js/react/components/accounts.js

```diff
@@ -74,51 +72,40 @@ const AccountList = React.createClass({
     }
 });
 
-const AccountRow = React.createClass({
+const AccountWrapper = React.createClass({
     propTypes: {
         account: React.PropTypes.object,
         activeAccount: React.PropTypes.object,
+        children: React.PropTypes.oneOfType([
```

Comment:
Just `React.PropTypes.node` should be enough here; an array of nodes is still a node.